tests/jwt_test.py

```diff
@@ -487,24 +487,13 @@ def check_kv_jwt_keys_not_empty(args, network, issuer):
     assert False, "No keys for issuer"
 
 
-def get_jwt_refresh_endpoint_metrics(network) -> dict:
-    primary, _ = network.find_nodes()
-    with primary.client(network.consortium.get_any_active_member().local_id) as c:
-        r = c.get("/node/api/metrics")
-        m = next(
-            v
-            for v in r.body.json()["metrics"]
-            if v["path"] == "jwt_keys/refresh" and v["method"] == "POST"
-        )
-        assert m["errors"] == 0, m["errors"]  # not used in jwt refresh endpoint
-        m["successes"] = m["calls"] - m["failures"]
-        return m
-
-
-def get_jwt_metrics(network) -> dict:
-    primary, _ = network.find_nodes()
-    with primary.client(network.consortium.get_any_active_member().local_id) as c:
-        r = c.get("/node/jwt_metrics")
+def get_jwt_refresh_endpoint_metrics(primary) -> dict:
```

Comment:
No, we're (for now) keeping the old-style metrics (ie - per-node state, updated during processing of each call), but _only_ for the `/jwt_keys/refresh` method. And now driven in a different way (instead of a `increment_metrics_X` call for every endpoint, we respond to the generic `request_completed` event in the node frontend to say "if this is `/jwt_keys/refresh`, count it"). Why? Because this test still uses it to confirm when/how calls to `/jwt_keys/refresh` completed, and it's hard to see that otherwise (this is an internal call, so we don't get to see any errors client side, so I think its fine to retain metrics for it).